transform/mattermost-analytics/models/marts/sales/hightouch/fct_marketing_campaign_member.sql

```diff
@@ -0,0 +1,38 @@
+with marketing as (
```

Comment:
Let's create a `marketing` mart and move this model there. 

Note that this might happen for other models as well, but we can fix it in a separate PR.

Reply:
Moved to new `marketing` folder